src/test/scala/cognite/spark/v1/TimeSeriesRelationTest.scala

```diff
@@ -34,6 +44,7 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
       .option("type", "timeseries")
       .option("collectMetrics", "true")
       .option("metricsPrefix", metricsPrefix)
+      .option("partitions", "1")
```

Comment:
Shouldn't it work without this as well?

Reply:
Absolutely, this is to make sure it works _with_ it!